langchain4j-core/src/main/java/dev/langchain4j/rag/DefaultRetrievalAugmentor.java

```diff
@@ -227,62 +225,69 @@ private static List<Content> retrieve(ContentRetriever retriever, Query query) {
     }
 
     private static Map<Query, Collection<List<Content>>> join(
-            Map<Query, CompletableFuture<Collection<List<Content>>>> queryToFutureContents) {
+        Map<Query, CompletableFuture<Collection<List<Content>>>> queryToFutureContents) {
         return allOf(queryToFutureContents.values().toArray(new CompletableFuture[0]))
-                .thenApply(ignored ->
-                        queryToFutureContents.entrySet().stream()
-                                .collect(toMap(
-                                        Map.Entry::getKey,
-                                        entry -> entry.getValue().join()
-                                ))
-                ).join();
+            .thenApply(ignored ->
+                queryToFutureContents.entrySet().stream()
+                    .collect(toMap(
+                        Map.Entry::getKey,
+                        entry -> entry.getValue().join()
+                    ))
+            ).join();
     }
 
     private static void logQueries(Query originalQuery, Collection<Query> queries) {
         if (queries.size() == 1) {
             Query transformedQuery = queries.iterator().next();
             if (!transformedQuery.equals(originalQuery)) {
                 log.debug("Transformed original query '{}' into '{}'",
-                        originalQuery.text(), transformedQuery.text());
+                    originalQuery.text(), transformedQuery.text());
             }
-        } else {
+        } else if (log.isDebugEnabled()){
```

Comment:
if we started updating logging, maybe we can wrap a whole content of this method with `if (log.isDebugEnabled())`. Looks all it does is logs with debug level

Reply:
Let's stop updating logging in this PR. Let's do it separately